app/org/sagebionetworks/bridge/dynamodb/DynamoInitializer.java

```diff
@@ -79,59 +80,49 @@ public static void init(String dynamoPackage) {
     }
 
     @SafeVarargs
-    public static void init(Class<? extends DynamoTable>... dynamoTables) {
-        List<Class<?>> classes = new ArrayList<Class<?>>();
-        for (Class<?> dynamoTable : dynamoTables) {
-            classes.add(dynamoTable);
-        }
+    public static void init(Class<?>... dynamoTables) {
+        List<Class<?>> classes = Arrays.asList(dynamoTables);
         List<TableDescription> tables = getAnnotatedTables(classes);
         initTables(tables);
     }
 
     /**
-     * For phasing out obsolete schemas.
+     * Actions performed before init(), e.g. for phasing out obsolete schemas.
      */
     static void beforeInit() {
     }
 
-    static void deleteTable(String table) {
-        table = getTableName(table);
+    static void deleteTable(Class<?> table) {
+        final String tableName = TableNameOverrideFactory.getTableNameOverride(table).getTableName();
```

Comment:
TableNameOverrideFactory.getTableNameOverride(table).getTableName() seems to come up a lot. Consider making a DynamoUtils class with this as a helper method. (At the very least, it might be easier to mock and test this way.)